README.md

```diff
@@ -36,6 +36,17 @@ The tutorial will create a Kubernetes Engine cluster that has a sample applicati
 
 ## Prerequisites
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```

Comment:
`[Google Cloud Shell][10] needs to be changed to [Google Cloud Shell](https://cloud.google.com/shell/docs/)`

Reply:
please test this